src/opm/input/eclipse/Schedule/Action/ActionAST.cpp

```diff
@@ -47,7 +47,7 @@ AST AST::serializeObject()
 }
 
 Action::Result AST::eval(const Action::Context& context) const {
-    if (this->condition)
+    if (!this->condition || this->condition->empty())
         return this->condition->eval(context);
     else
         return Action::Result(false);
```

Comment:
Ehm, you probably meant to either switch the `return` statements or reverse the branch condition here.

Reply:
Ehmmm - yes!